### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by spdustin** [spdustin/ChatGPT-AutoExpert#6] [Announcements]
Heya, folks! I thought I’d open up discussions for anyone who has questions or comments that don’t seem like they’d be a good fit for Issues.

**Comment by darkviruzz** [comment]
Hi Dustin, Thank you for this gem!
i really like the nuanced way of the standard edition (though i completely removed the link-generation) and i love the /memory function if i use ADA. but it's really a pain to always have to change the instructions per chat...
do you think it would be possible to merge the best of both worlds? as i see the character limit of about me and CI of course it will not go without leaving stuff out. What do you think?

**Reply by spdustin** [reply]
I'm working on a Chrome companion extension to switch between AutoExpert prompts easily, and it will include a "builder" to pick-and-choose which features you want for each edition. Do you think that would be helpful for your use case?

**Reply by panckreous** [reply]
Firefox too please

**Reply by spdustin** [reply]
Roger that, @panckreous. Safari too, hopefully one that works with iOS. At the moment, I’ve been prototyping with UserScripts, so it shouldn’t be too tough to automate a build of plugins for the big 3.

**Reply by TheNoteTaker** [reply]
Hey man you got a discord? Shoot me a request: amh5954

**Comment by Tizzz-555** [comment]
Hi Dustin, thanks for this great idea, I am having fun with it! I am not sure I understood one thing though: after we open the first new chat and launch the autodev.py file, do we still need to relaunch that file for every new chat we open if we want AutoExpert to be used for that chat? 
Would the idea be to use it like this or to just have a very long chat named AutoExpert for working on all coding related projects?

**Reply by spdustin** [reply]
You’d re-upload autodev.py on each new chat, plus a zipped memory file (if you made one)

**Comment by C-Pearson31** [comment]
@spdustin I am loving your AutoExpertv5! Can you provide some details on how non-paid ChatGPT users can incorporate this?

**Reply by spdustin** [reply]
Thanks for writing! You can use the GPT-3.5 instructions for AutoExpert Standard Edition, but the Developer Edition replies on a companion file for some of its functionality, which limits it to GPT-4 with Advanced Data Analysis. It’ll still _kinda_ work in GPT-3.5, but honestly, I wouldn’t trust the code output of 3.5 for anything complicated.

**Comment by Abdulqyuom** [comment]
Abdulqyuom

**Comment by Abdulqyuom** [comment]
PayPal account

**Comment by ericrascon** [comment]
Hey, Dustin.

Loving the AutoExperctv5! Checked out your articles on Medium and it feels like you're one of the few ones who know how prompts really work.

Question. All the other systems prompts for DALLE and Bing are they supposed to be placed in in the "How would you like ChatGPT to respond?" I ask because they're over the character limit - don't know if I'm doing some wrong or not?

**Reply by spdustin** [reply]
Sorry for missing this. Those are the system prompts used by different modes of ChatGPT, not custom instructions. They're used "behind the scenes" in addition to your custom instructions.

**Comment by C-Pearson31** [comment]
Is it just me or is ChatGPT ignoring AutoExpert’s CI lately?

I like the new bot but miss the ability to get the markdown table,
re-stated problem, etc that I got used to with v5.


~ CP

**Reply by spdustin** [reply]
This has been idle for a while. I'm working on updates to the custom instructions to align with the new features in the [custom GPT](https://chat.openai.com/g/g-LQHhJCXhW-autoexpert-chat).

**Comment by C-Pearson31** [comment]
Sweet! I have been wondering what you were up to and eagerly awaiting your
next update.


~ CP

**Comment by zlocorp** [comment]
Hi, I was using version 5 and now I've upgraded to version 6, should I delete the settings I made before in ChatGPT to use version 6 correctly?

**Comment by whh1206** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by Jhkok5** [comment]
Hi, first of all, thanks for your great work! 

Unfortunately AutoExpert stopped working for me as of yesterday. Chatgpt reports that the GPT was not found when i open it from the sidebar or from a direct link.
